docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc

```diff
@@ -105,7 +95,6 @@ ifeval::["{build}" != "upstream"]
 For more information on deploying the clustered Ceph NFS service, see the
 link:https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/7/html-single/operations_guide/index#management-of-nfs-ganesha-gateway-using-the-ceph-orchestrator[Management of NFS-Ganesha gateway using the Ceph Orchestrator (Limited Availability)] in _Red Hat Ceph Storage 7 Operations Guide_.
 endif::[]
-//kgilliga: Confirm that we should link to the Ceph Operations Guide downstream.
 * The following commands are run inside a `cephadm shell` to create a clustered
 Ceph NFS service.
 +
```

Comment:
Why is it relevant to mention that the “following commands are run inside the cephadm shell” here? Should these commands be steps?

Reply:
I agree the following content should be rewritten under form of steps (I assume it makes the style consistent with the rest of the documentation).
In the snippet below, you can use the comments to build the statement related to the step, and then put the command associated.

Reply:
Also remove "{{ cluster size }}" from the `ceph nfs cluster create ` command.